docs/index.html

```diff
@@ -1,7 +1,4 @@
-<!doctype html>
-<html lang="en">
-
-<head>
+<html lang="en"><head>
```

Comment:
why is this being changed to drop standards? 

[Atomic commits](https://en.wikipedia.org/wiki/Atomic_commit) please.